src/components/Utils/formFieldValidators.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash';
+
+export const combineValidators = (validators) => (...args) => validators
```

Comment:
Are you sure we need this? redux-form supports already multiple validation functions, https://redux-form.com/7.4.2/examples/fieldlevelvalidation/

Reply:
Thanks, good point to revise redux-form. Fixed